src/rgw/rgw_s3select.cc

```diff
@@ -547,8 +551,7 @@ int RGWSelectObj_ObjStore_S3::run_s3select_on_json(const char* query, const char
   }
     
   //initializing json processor
-  json_object::csv_definitions output_definition;
-  m_s3_json_object.set_json_query(&s3select_syntax,output_definition);
+  m_s3_json_object.set_json_query(&s3select_syntax, json);
```

Comment:
why did you move the csv definitions. does not seem to be used before this line?

Reply:
this change is related to other PR (JSON output format. user may request a JSON format per SQL result),
that PR requires a change to the `json_object::csv_definitions` (output format definition).

for this PR it is not relevant (it just reduced differences between PR's)